command/harness/dlite/dlite.go

```diff
@@ -50,7 +51,12 @@ func (c *dliteCommand) startPoller(ctx context.Context, tags []string) error {
 	// Client to interact with the harness server
 	client := delegate.New(c.env.Dlite.ManagerEndpoint, c.env.Dlite.AccountID, c.env.Dlite.AccountSecret, true)
 	p := poller.New(c.env.Dlite.AccountID, c.env.Dlite.AccountSecret, c.env.Dlite.Name, tags, client, r)
-	err := p.Poll(ctx, taskExecutors, taskInterval)
+	info, err := p.Register(ctx)
+	if err != nil {
+		return err
```

Comment:
log the error

Reply:
logged in response of start poller on L109